README.md

```diff
@@ -16,37 +11,28 @@ artifacts available @ [*bintray repositories*](https://bintray.com/ticktok-io/ma
 ```
 dependencies {
   ...
-    compile 'io.ticktok:ticktok-java-client:<version>'
+    compile 'io.ticktok:ticktok-java-client:0.3.0'
```

Comment:
should be generic, this is why i added a link to the bintray

Reply:
it's pretty standard to specify latest version:
Few examples:
https://github.com/rabbitmq/rabbitmq-java-client
https://github.com/perwendel/spark
https://github.com/xetorthio/jedis

Reply:
yes, but still i dont want to maintain.
really easy to follow the link...

Reply:
Can't argue with that